scripts/dts/extract_dts_includes.py

```diff
@@ -60,7 +60,9 @@ def bindings(cls, compatibles, yaml_dirs):
                 for filename in fnmatch.filter(filenames, '*.yaml'):
                     cls._files.append(os.path.join(root, filename))
 
-        yaml_list = {}
+        yaml_list['node'] = {}
+        yaml_list['bus'] = {}
+        yaml_list['compat'] = []
```

Comment:
Hmm... this confuses me though. Doesn't `yaml_list` just map compats (from lines like `constraint: "nxp,imx-lpspi"`) to the parsed yaml from that file?

Would it make sense to have these as separate variables, instead of as keys in `yaml_list`/`compat_to_yaml`?

Reply:
Renaming yaml_list to `bindings` would make the intention a little bit clearer.

Reply:
agreed

Reply:
Fine with me